_pages/en_US/uninstall-cfw.txt

```diff
@@ -26,7 +26,16 @@ Note that if you have any payload files other than `GodMode9.firm` in the `/luma
 
 ### Instructions
 
-#### Section I - Safety Test
+#### Section I - Checking for Flipnote
+1. Open system settings
+1. Click Internet
+1. Click DS Conncetion Settings
```

Comment:
The guide tends to use the term "navigate" instead of "click" for things on the 3DS (but not always). This should be one step

Reply:
I will fix all of these in a sec